types/src/transaction/authenticator.rs

```diff
@@ -240,6 +307,15 @@ pub enum Scheme {
     MultiEd25519 = 1,
     // ... add more schemes here
 }
+impl Scheme {
+    fn from_u8(value: u8) -> Scheme {
+        match value {
+            0 => Scheme::Ed25519,
+            1 => Scheme::MultiEd25519,
+            _ => panic!("Unknown value: {}", value),
```

Comment:
should not panic, this cause a client can crash server  by submitting a transaction with an invalid scheme number.